src/ray/raylet/worker_pool.cc

```diff
@@ -76,7 +78,10 @@ WorkerPool::WorkerPool(instrumented_io_context &io_service, const NodeID node_id
           num_initial_python_workers_for_first_job, maximum_startup_concurrency)),
       num_initial_python_workers_for_first_job_(num_initial_python_workers_for_first_job),
       periodical_runner_(io_service),
-      get_time_(get_time) {
+      get_time_(get_time),
+      worker_process_in_container_(worker_process_in_container),
+      temp_dir_(temp_dir),
+      session_dir_(session_dir) {
```

Comment:
```suggestion
      session_dir_(std::move(session_dir)) {
```
or pass it as a reference.